run.sh

```diff
@@ -5,12 +5,12 @@ python3 run_data_measurements.py --dataset="hate_speech18" --config="default" --
 python3 run_data_measurements.py --dataset="hate_speech_offensive" --config="default" --split="train" --label_field="label" --feature="tweet"
```

Comment:
Why not add --overwrite previous to all of these?

Reply:
Can do!